scenes/scene.yaml

```diff
@@ -6,10 +6,10 @@ fonts:
         url: https://fonts.gstatic.com/s/montserrat/v7/zhcz-_WihjSQC0oHJ9TCYL3hpw3pgy2gAi-Ip7WPMi0.woff
     Open Sans:
         - weight: 300
-          url: fonts/open sans-300normal.ttf
+          url: https://github.com/tangrams/bubble-wrap/raw/gh-pages/fonts/OpenSans-Regular.woff
```

Comment:
For Mapzen house styles we usually add a comment with the remote URL and follow that with an uncommented local resources:

``` yaml
#url: https://fonts.gstatic.com/s/montserrat.woff
url: fonts/montserrat.woff
```